docs/dotnet-five.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What is .NET 5
+description: In this article you will learn about .NET 5, a cross platform, and open-source project that convergences .NET framework and .NET Core together.
+ms.date: 08/31/2020
+ms.topic: overview
+ms.author: dapine
+author: IEvangelist
+---
+
+# What is .NET 5
+
+The .NET ecosystem is being simplified with the convergence of .NET Core and .NET Framework. Naming things is hard, but unifying versions of two disparate .NET implementations can be harder. This article details .NET 5, which can be thought of as .NET Core vNext. With .NET Core concluding at version 3.1 and .NET Framework at version 4.8, to avoid ambiguity the next logical version is .NET version 5.0.
+
+## Evolution of .NET
+
+The advent of .NET Core has evolved the .NET ecosystem in compelling ways. It matured as an open-source project on GitHub, celebrating community contributions, and humbly improving over time. With .NET 5, you get all of the things you loved about .NET Core:
+
+> [!div class="checklist"]
+>
+> - Cross-platform
+> - Open-source and community-oriented
+> - Side-by-side installation
+> - Small project files (SDK-style)
+> - High performance
+> - Command-line tooling
+> - Visual Studio, Visual Studio for Mac, and Visual Studio Code integration
+> - Flexible deployment
+
+In addition to all of these key characteristics, .NET 5 is making significant [performance improvements](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-5).
+
+## .NET release schedule
+
+.NET has a formal release schedule, where a new major version is planned every November. .NET 5 will not have long-term support (LTS), however; .NET 6 in 2021 will be LTS. As major .NET versions ship, even-numbered releases will receive LTS. Minor releases will occur as needed.
+
+:::image type="content" source="media/dotnet-schedule.svg" alt-text=".NET release schedule":::
+
+## What happens to .NET Standard
+
+With .NET 5, .NET Standard is being de-emphasized. If you're planning on sharing code between .NET Framework, .NET Core, and .NET 5 workloads - you can do so by specifying `netstandard2.0` as your target framework moniker (TFM). For more information, see [How to specify target frameworks](standard/frameworks.md#how-to-specify-target-frameworks).
```

Comment:
This feels odd to me. I expected more clarification about _why_ it is being de-emphasized, but then the rest of the paragraph talks about how to still use it.

Reply:
It is my understanding the it's being de-emphasized, as you'll only have to use `net5.0` as your TFM for all project types, right? I can update that.

Reply:
> for all project types

Wouldn't libraries still have to target standard to be usable with .NET 5 and .NET Framework?